.github/renovate.json5

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboardApproval"
+    ],
+    "dependencyDashboard": true,
+    "dependencyDashboardLabels": ["dependencies", "repo maintenance"],
+    "enabledManagers": ["github-actions", "npm"],
+    "internalChecksFilter": "strict",
+    "labels": ["dependencies"],
+
+    // Wait a few days for any new package, as a precaution against malicious publishes
+    "minimumReleaseAge": "3 days",
+
+    "packageRules": [
+        // Enable major version updates, but don't auto-merge them
+        {
+            "automerge": false,
+            "matchUpdateTypes": ["minor"]
+        },
+
+        // Enable minor version updates with auto-merging
+        {
+            "automerge": true,
+            "automergeStrategy": "squash",
+            "matchUpdateTypes": ["minor"]
+        },
+
+        // Ignore patch versions to reduce noise
+        {
+            "enabled": false,
+            "matchUpdateTypes": ["patch"]
+        },
+
+        // Label PRs appropriately per their manager
+        {
+            "addLabels": ["javascript"],
+            "matchManagers": ["npm"]
+        },
+        {
+            "addLabels": ["github-actions"],
+            "matchManagers": ["github-actions"]
+        },
+
+        // Batch package sets together
+        {
+            "groupName": "babel",
+            "matchPackagePrefixes": ["@babel", "babel-"]
+        },
+        {
+            "groupName": "wdio",
+            "matchPackagePrefixes": ["@wdio"]
+        },
+        {
+            "groupName": "metascraper",
+            "matchPackagePrefixes": ["metascraper"]
+        }
+    ],
+
+    // By default renovate will auto-rebase whenever the dep branch falls behind main.
+    // This is annoying as it spams notifications and creates unnecessary action runs.
+    // Instead only auto-rebase when conflicted, and we can trigger a manual rebase if required.
+    "rebaseWhen": "conflicted"
```

Comment:
The current config will automerge `minor` updates:

> ```
>         // Enable minor version updates with auto-merging
>         {
>             "automerge": true,
>             "automergeStrategy": "squash",
>             "matchUpdateTypes": ["minor"]
>         },
> ```

Setting `rebaseWhen=conflicted` may cause problems when automerging! I'll quote the full `rebaseWhen` docs: [^1]

> ## rebaseWhen
> 
> Possible values and meanings:
> 
> - `auto`: Renovate will autodetect the best setting. It will use `behind-base-branch` if configured to automerge or repository has been set to require PRs to be up to date. Otherwise, `conflicted` will be used instead
> - `never`: Renovate will never rebase the branch or update it unless manually requested
> - `conflicted`: Renovate will rebase only if the branch is conflicted
> - `behind-base-branch`: Renovate will rebase whenever the branch falls 1 or more commit behind its base branch
> 
> `rebaseWhen=conflicted` is not recommended if you have enabled Renovate automerge, because:
> 
> - It could result in a broken base branch if two updates are merged one after another without testing the new versions together
> - If you have enforced that PRs must be up-to-date before merging (e.g. using branch protection on GitHub), then automerge won't be possible as soon as a PR gets out-of-date but remains non-conflicted
> 
> It is also recommended to avoid `rebaseWhen=never` as it can result in conflicted branches with outdated PR descriptions and/or status checks.
> 
> Avoid setting `rebaseWhen=never` and then also setting `prCreation=not-pending` as this can prevent creation of PRs.

Did you read the [Noise reduction docs](https://docs.renovatebot.com/noise-reduction/)? That has ways to reduce noise, I'll summarize them here:

- Package grouping
- Scheduling when Renovate runs
- Automerging
- Lock file considerations

[^1]: [Renovate docs for `rebaseWhen`](https://docs.renovatebot.com/configuration-options/#rebasewhen)

Reply:
👍 thanks, removing.